GamestonkTerminal/GamestonkTerminal
# Create a packaged app bundle with Pyinstaller

This PR adds the infrastructure required to build the terminal into an executable using pyinstaller.

TODOS:
- [x] Test if it's working on Windows
- [x] Test if a separate bundle is required to use the prediction menu

---

How to create the bundle:

- activate the gst environment
- run `poetry install -E installer` to install the `pyinstaller` package into the environment
- run `pyinstaller build/pyinstaller/terminal.spec` from the project root

As the result you should have a `terminal` executable appear in the dist folder
